frontend/webpack.config.js

```diff
@@ -6,7 +6,7 @@ import path from 'path';
 import { fileURLToPath } from 'url';
 
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
-const env = dotenv.config().parsed;
+const env = { ...process.env, ...dotenv.config().parsed };
```

Comment:
this change has been added as we need the front-end to use configuration from any available environment variables (passed into the docker container). Previously, it relied on a local `.env` file, which may no longer exist. Open to suggestions on better ways to achieve this